### PR TITLE
[docs] Update standalone compiler docs

### PR DESCRIPTION
**Pull request by kuhar**
List the most common pipeline compilation options.
Remove an obsolete option (`--entry-target`).
Also minor formatting fixes.

### CODE REVIEW

File: llpc/docs/amdllpc.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
 # amdllpc Standalone Compiler
 
-LLPC can be built into a standalone offline compiler (amdllpc). It supports GLSL, SPIR-V binary and SPIR-V assemble file as input and output GPU ISA code and related register settings.
+LLPC can be built into a standalone offline compiler (amdllpc). It supports GLSL, SPIR-V binary and SPIR-V assembly file as input and outputs GPU ISA code and related register settings (PAL metadata).
 
-## Build Instruction
+## Build instruction
```
Comment (vettoreldaniele) [inline llpc/docs/amdllpc.md:5]: nit: instructions?

File: llpc/docs/amdllpc.md [hunk 2]
```diff
@@ -51,12 +54,12 @@ cmake --build build --target check-lgc check-lgc-units check-amdllpc check-amdll
 ```
 
 See above if this gives an error due to not finding an include file from glslang or SPIRV-Tools.
-If you want to make the amdllpc compatible with driver, you could get <pal_interface_version> from the ICD_PAL_CLIENT_MAJOR_VERSION defined in xgl/icd/make/importdefs and add it in the build option. If the build option is not added, latest PAL interface version will be used.
+If you want to make the amdllpc compatible with driver, you could get `<pal_interface_version>` from the `ICD_PAL_CLIENT_MAJOR_VERSION` defined in `xgl/icd/make/importdefs` and add it in the build option. If the build option is not added, latest PAL interface version will be used.
```
Comment (vettoreldaniele) [inline llpc/docs/amdllpc.md:57]: nit: "make the amdllpc" -> "make amdllpc"

"add it in the build option" -> "add it to the build options". 

"latest PAL" -> "the latest PAL"

**Comment by kuhar** [comment]
V1: Minor grammar fixes.

I pasted the whole document to google docs and did not find any remaining blue squiggles.

**Comment by amdvlk-admin** [comment]
### Test summary for commit 98c26b1f29f909a323e268900bde1c833b67cf84
<details><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master 39f8940199e60c44d4211cf8165dfd12876316fa</li>
<li>METROHASH: amd-master 3c566dd9cda44ca7fd97659e0b53ac953f9037d2</li>
<li>PAL: dev 0a0a4ae4ab062d31fcd68863c35952967a8060ee</li>
<li>SPVGEN: dev 6c2f36bcf0f8ff7ba8d20c8b789ca6346d1e16b1</li>
<li>XGL: dev c41a825d1ea17c108e8fb123f082357f849e7823</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev c20ba7614ecdc78b3d99601423783100835200c1</li>
</ul></details>

<details><summary>CTS tests (Failed: 0/228192)</summary><ul><li>Built with version 1.2.5.2</li>
<details open><summary>Rhel 8.2, Gfx10</summary><ul>

<li>Passed: 33955/57048 (59.5%)</li>
<li>Failed: 0/57048 (0.0%)</li>
<li>Not Supported: 23093/57048 (40.5%)</li><li>Warnings: 0/57048 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Passed: 33797/57048 (59.2%)</li>
<li>Failed: 0/57048 (0.0%)</li>
<li>Not Supported: 23250/57048 (40.8%)</li><li>Warnings: 1/57048 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx8</summary><ul>

<li>Passed: 34085/57048 (59.7%)</li>
<li>Failed: 0/57048 (0.0%)</li>
<li>Not Supported: 22963/57048 (40.3%)</li><li>Warnings: 0/57048 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx103</summary><ul>

<li>Passed: 35882/57048 (62.9%)</li>
<li>Failed: 0/57048 (0.0%)</li>
<li>Not Supported: 21166/57048 (37.1%)</li><li>Warnings: 0/57048 (0.0%)</li>
</ul></details>


</ul></details>

**Review by vettoreldaniele — APPROVED** [review]

**Review by amdrexu — APPROVED** [review]

**Comment by kuhar** [comment]
retest this please

**Comment by amdvlk-admin** [comment]
### Test summary for commit 98c26b1f29f909a323e268900bde1c833b67cf84
<details><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master 39f8940199e60c44d4211cf8165dfd12876316fa</li>
<li>METROHASH: amd-master 3c566dd9cda44ca7fd97659e0b53ac953f9037d2</li>
<li>PAL: dev 0a0a4ae4ab062d31fcd68863c35952967a8060ee</li>
<li>SPVGEN: dev 6c2f36bcf0f8ff7ba8d20c8b789ca6346d1e16b1</li>
<li>XGL: dev c41a825d1ea17c108e8fb123f082357f849e7823</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev c20ba7614ecdc78b3d99601423783100835200c1</li>
</ul></details>

<details><summary>CTS tests (Failed: 0/228192)</summary><ul><li>Built with version 1.2.5.2</li>
<details open><summary>Rhel 8.2, Gfx10</summary><ul>

<li>Passed: 33955/57048 (59.5%)</li>
<li>Failed: 0/57048 (0.0%)</li>
<li>Not Supported: 23093/57048 (40.5%)</li><li>Warnings: 0/57048 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Passed: 33797/57048 (59.2%)</li>
<li>Failed: 0/57048 (0.0%)</li>
<li>Not Supported: 23250/57048 (40.8%)</li><li>Warnings: 1/57048 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx8</summary><ul>

<li>Passed: 34085/57048 (59.7%)</li>
<li>Failed: 0/57048 (0.0%)</li>
<li>Not Supported: 22963/57048 (40.3%)</li><li>Warnings: 0/57048 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx103</summary><ul>

<li>Passed: 35882/57048 (62.9%)</li>
<li>Failed: 0/57048 (0.0%)</li>
<li>Not Supported: 21166/57048 (37.1%)</li><li>Warnings: 0/57048 (0.0%)</li>
</ul></details>


</ul></details>

**Pull request merged**